spock-core/src/main/groovy/spock/util/EmbeddedSpecCompiler.groovy

```diff
@@ -110,11 +112,12 @@ class EmbeddedSpecCompiler {
     }
 
     loader.loadedClasses.findAll {
-      SpecUtil.isSpec(it) || isJUnitTest(it) // need JUnit tests sometimes
+      SpecUtil.isSpec(it) || isJUnit4Test(it) // need JUnit tests sometimes
```

Comment:
do we really need junit4 tests with the EmbeddedSpecCompiler?

Reply:
It's not internally used by the Spock test suite. Most likely not many people use it. I removed it